### PR TITLE
Create lua.lua

### PR DESCRIPTION
**Pull request by ItsTato**
was originally gonna write a py.lua but I saw someone else did already so instead I settled on making lua.lua after I realized our bussin editor didn't have it integrated

### CODE REVIEW

File: Lua/Plugins/lua.lua [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+hlreg("'","'","string",false);
+hlreg("\"","\"","string",false);
+hlreg("[[","]]","string",false);
+
+hlreg("--","","comments",true);
+hlreg("--[[","]]","comments",false);
+
+-- 💀🔥🗣️👍
+add_comment("lego game language");
```
Comment (face-hh) [inline Lua/Plugins/lua.lua:65]: LMFAO

**Comment by face-hh** [comment]
Didn't even know this was possible with the Lua API!
image

You've done an amazing job :O

**Comment by ItsTato** [comment]
Thank you! :D
github just suggested me to use a dagger emoji after typing in :D, lmao

btw, if you type in, for example, **os.c** and select **os.clock** the editor adds **os.os.clock** instead of **os.clock**.
it functions perfectly if you just type in **osc** and select **os.clock** from there though. 

should i go ahead and open up a nice lil issue 4 this?

**Comment by face-hh** [comment]
Yup, you should open an issue on it

**Comment by ItsTato** [comment]
alr, will do!

**Pull request merged**